### PR TITLE
Fix modbus switch example configuration

### PR DESCRIPTION
**Pull request by llytvynenko**
The error in the documentation prevents newcomers from an easy start.
Please see https://community.home-assistant.io/t/modbus-component-cant-setup-platform/151156 as an example of misconfiguration that might happen because of the error in the docs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switch.modbus.markdown [hunk 1]
```diff
@@ -24,6 +24,7 @@ switch:
       slave: 1
       coil: 13
     - name: Switch2
+      hub: hub1
```
Comment (frenck) [inline source/_integrations/switch.modbus.markdown:27]: Hub isn't a required option, and actually should be removed from all examples in the first example.

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/switch.modbus.markdown (hunk 1 above)
Comment (llytvynenko) [reply source/_integrations/switch.modbus.markdown:27]: While hub might not be required in general, it is definitely required for `modbus`, I've had this issue - that's why I've created the pull request.
If you don't set hub name you will get [configuration error](https://community.home-assistant.io/t/modbus-component-cant-setup-platform/151156).

Comment (frenck) [reply source/_integrations/switch.modbus.markdown:27]: According to the integration code, `hub` is optional and has a default value: <https://github.com/home-assistant/core/blob/dev/homeassistant/components/modbus/__init__.py#L83>

The documentation should reflect the integration code. If the integration handles it incorrect, that should be fixed. The documentation should not be used as a workaround.

Comment (llytvynenko) [reply source/_integrations/switch.modbus.markdown:27]: I'm not sure I'll be able to jump into the code and figure what's wrong, hopefully someone will jump in:)
But I totally agree with your points, let's discard this pull request than!

**Pull request closed without merging**